cloudhub/modules/ROOT/pages/secure-application-properties.adoc

```diff
@@ -117,7 +117,7 @@ After you set the property, you can't retrieve it; however, you can overwrite th
 To update a value, enter a new value into the field.
 
 After an application is deployed with safely hidden application properties, CloudHub maintains the security flag for those properties.
-Even if you edit your application file to remove the `secureProperties` definitions from your `mule-artifact.json` file and then upload that edited application file to CloudHub, CloudHub maintains the invisibility of the properties previously flagged as safely hidden.
+Even if you edit your application file to remove the `secureProperties` definitions from your `mule-artifact.json` file and then upload that edited application file to CloudHub, CloudHub maintains the invisibility of the properties previously flagged as safely hidden. Any property added as secure at anytime remains as secure even after it is deleted from the `mule-artifact.json` file.
```

Comment:
[Suggestion] and typo fix (anytime -> any time):
`Even if you edit your application file to remove the `secureProperties` definitions from your `mule-artifact.json` file and then upload that edited application file to CloudHub, CloudHub [maintains [-> keeps]] the invisibility of the properties previously flagged as safely hidden. Any property added as secure at any time remains [remove 'as'] secure [[-> hidden? forever?]] even after it is deleted from the `mule-artifact.json` file.`